pkg/sql/sqlbase/system.go

```diff
@@ -567,16 +565,16 @@ var (
 			},
 		},
 		NextIndexID:    3,
-		Privileges:     NewPrivilegeDescriptor(security.RootUser, SystemDesiredPrivileges(keys.JobsTableID)),
+		Privileges:     NewCustomRootPrivilegeDescriptor(SystemDesiredPrivileges(keys.JobsTableID)),
 		FormatVersion:  InterleavedFormatVersion,
 		NextMutationID: 1,
 	}
 
 	// WebSessions table to authenticate sessions over stateless connections.
 	WebSessionsTable = TableDescriptor{
 		Name:     "web_sessions",
-		ID:       19,
-		ParentID: 1,
+		ID:       keys.WebSessionsTableID,
```

Comment:
this table seemed to be doing this a bit more manually, it now follows the pattern used by other tables.